api/rpc/logs/logs.go

```diff
@@ -20,8 +20,8 @@ import (
 
 // Server is used to implement log.LogServer
 type Server struct {
-	Es            *elasticsearch.Elasticsearch
-	NatsStreaming *ns.NatsStreaming
+	Es *elasticsearch.Elasticsearch
+	Ns *ns.NatsStreaming
```

Comment:
Bike-shedding a bit here, but consider `Ns` -> `NS` (and `Es` to `ES` while you're at it (see [Initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)).